ports/mimxrt/Makefile

```diff
@@ -397,6 +397,11 @@ CFLAGS += \
 	-DFSL_FEATURE_PHYKSZ8081_USE_RMII50M_MODE=1
 endif
 
+ifdef MICROPY_HW_FLASH_CLK
+	CFLAGS += \
+			-DMICROPY_HW_FLASH_CLK=$(MICROPY_HW_FLASH_CLK)
+endif
```

Comment:
Would it be simpler to define this config in the board's `mpconfigboard.h` file, so this bit is not needed?

In general I would rather have things defined at the C level rather than the makefile level.

Reply:
> Would it be simpler to define this config in the board's `mpconfigboard.h` file

It would, but all other flash config options are defined in `mpconfigboard.mk` files, so this new option belongs there (at least for now). Generally speaking, for this port we're just following the existing patterns to avoid major refactoring.

https://github.com/micropython/micropython/blob/master/ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk#L6

Reply:
OK, sounds good.

Can you please put the CFLAGS on one line, ie don't use the backslash.